src/gui/hud/SecondaryInventory.cpp

```diff
@@ -558,7 +558,8 @@ bool SecondaryInventoryHud::dragEntity(Entity * io, const Vec2s & pos) {
 		}
 	}
 	
-	return false;
+	Set_DragInter(io);
```

Comment:
Thanks.

Btw, how would you like to be credited in the [AUTHORS file](https://github.com/arx/ArxLibertatis/blob/master/AUTHORS) - just `bsxf-47`?